homework7.playground/Contents.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import Foundation
+
+class School {
+    var students = [Student]()
+    
+    init(students: [Student] = [firstStudent, secondStudent, thirdStudent, fourthStudent]) {
+        self.students = students
+    }
+    func printInfo() {
+        let info = [firstStudent, secondStudent, thirdStudent, fourthStudent]
+        print(info)
```

Comment:
You should use only `students` property inside the class. This function will stop working if you pass other data to `students`. You need to go through your `students` with cycle and call `printInfo()` in every iteration